packages/stylesheets/core/core.js

```diff
@@ -0,0 +1 @@
+import './scss/core-ingredients.scss';
```

Comment:
Love this one!

Reply:
Me too! This will save quite some blocking css in a few apps, without needing deep imports.